src/utils/export-image-utils.js

```diff
@@ -1,74 +0,0 @@
-// Copyright (c) 2020 Uber Technologies, Inc.
```

Comment:
why is this deleted?

Reply:
None of the methods was actually used. They were duplicated from export-utils.